js/static_dependencies/node-rsa/schemes/pkcs1.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * PKCS1 padding and signature scheme
+ */
+
+var BigInteger = require('../libs/jsbn');
+var crypt = require('crypto');
```

Comment:
The last reference to crypto, and the hardest one to cut out... ((

We already have two or three bignumber-libs ) `bignumber.js`, `bn.js`, now `jsbn` )

Reply:
removed it )

Reply:
We should reuse the bignumber lib that is installed and remove the need from BigNumber to be installed to use web3

Reply:
Weirdly enough if you install BigNumber.js it also install RSA sign by kenji %%<$ - look at the src inside node_modules if you don't believe me

Reply:
```
└── node_modules
    └── bignumber
        ├── LICENSE
        ├── README.md
        ├── example.js
        ├── index.js
        ├── lib
        │   └── rsa
        │       ├── b64.js
        │       ├── index.js
        │       ├── jsbn.js
        │       ├── rng.js
        │       ├── rsa.js
        │       └── sign.js
        ├── original
        │   ├── LICENSE
        │   ├── base64.js
        │   ├── jsbn.js
        │   ├── jsbn2.js
        │   ├── prng4.js
        │   ├── rng.js
        │   ├── rsa.js
        │   └── rsa2.js
        └── package.json
```

Reply:
and it also relies on jsbn at the end of the day...

coding is dumb, i quit.